app/models/author.rb

```diff
@@ -1,4 +1,4 @@
 class Author < ApplicationRecord
     has_and_belongs_to_many :books
-    validates :name, uniqueness:  true, allow_blank: false
+    validates :name, :uniqueness => {:case_sensitive => false}, allow_blank: false
```

Comment:
- still validate for uniqueness though?
- I think allow_blank is not working as per our discussion yesterday?

Reply:
- I didn't understand
- No error yesterday was because of (HTML && js) not database